tests/testDipole.py

```diff
@@ -80,10 +80,10 @@ def createDipole(w, h, xc, yc, scaling=100.0, fracOffset=1.2):
     offset = fracOffset * psfFwhmPix // 2
     array = image.getImage().getArray()
     xp, yp = xc - psfw//2 + offset, yc - psfh//2 + offset
-    array[yp:yp+psfh, xp:xp+psfw] += psfim.getArray()
+    array[int(yp):int(yp)+psfh, int(xp):int(xp)+psfw] += psfim.getArray()
```

Comment:
Well, can you intify `xp` and `yp` before using them as indices then so we have less clutter in the array indexing itself?